content/soarian/r4/foundation/conformance/operation-definition.md

```diff
@@ -0,0 +1,106 @@
+
+The following fields are returned if valued:
+
+* [Id](http://hl7.org/fhir/r4/resource-definitions.html#Resource.id){:target="_blank"}
+* [Name](http://hl7.org/fhir/r4/operationdefinition-definitions.html#OperationDefinition.name){:target="_blank"}
+* [Title](http://hl7.org/fhir/r4/operationdefinition-definitions.html#OperationDefinition.title){:target="_blank"}
+* [Status](http://hl7.org/fhir/r4/operationdefinition-definitions.html#OperationDefinition.status){:target="_blank"}
+* [URL](http://hl7.org/fhir/r4/operationdefinition-definitions.html#OperationDefinition.url){:target="_blank"}
+* [Kind](http://hl7.org/fhir/r4/operationdefinition-definitions.html#OperationDefinition.kind){:target="_blank"}
+* [Date](http://hl7.org/fhir/r4/operationdefinition-definitions.html#OperationDefinition.date){:target="_blank"}
+* [Publisher](http://hl7.org/fhir/r4/operationdefinition-definitions.html#OperationDefinition.publisher){:target="_blank"}
+* [Description](http://hl7.org/fhir/r4/operationdefinition-definitions.html#OperationDefinition.description){:target="_blank"}
+* [Jurisdiction](https://hl7.org/fhir/operationdefinition-definitions.html#OperationDefinition.jurisdiction){:target="_blank"}
+* [Code](http://hl7.org/fhir/r4/operationdefinition-definitions.html#OperationDefinition.code){:target="_blank"}
+* [Resource](http://hl7.org/fhir/r4/operationdefinition-definitions.html#OperationDefinition.resource){:target="_blank"}
+* [System](http://hl7.org/fhir/r4/operationdefinition-definitions.html#OperationDefinition.system){:target="_blank"}
+* [Type](http://hl7.org/fhir/r4/operationdefinition-definitions.html#OperationDefinition.type){:target="_blank"}
+* [Instance](http://hl7.org/fhir/r4/operationdefinition-definitions.html#OperationDefinition.instance){:target="_blank"}
+* [Parameter](http://hl7.org/fhir/r4/operationdefinition-definitions.html#OperationDefinition.parameter){:target="_blank"}
+
+## Search
+
+List an individual operation by its supplied parameter: 
+
+    GET /OperationDefinition?:parameters
+
+_Implementation Notes_
+
+* Authentication is not required to access the OperationDefinition resource
+
+### Authorization Types
+
+Authorization is not required.
+
+### Headers
+
+<%= headers head: {Accept: 'application/fhir+json, "application/json", "application/json+fhir"'} %>
+
+### Example
+
+
+#### Request
+
+    GET https://fhir-ehr-sc.cerner.com/r4/2f8f5ec1-b7b8-4be5-ae27-e308284dd9c1/OperationDefinition?url=https://fhir-ehr-sc.cerner.com/r4/OperationDefinition/group-export
+
+#### Response
+
+<%= headers status: 200 %>
+<%= json(:R4_OPERATION_DEFINITION_SEARCH) %>
+
+## Retrieve by id
```

Comment:
Change to Retrieve by ID.
PRODOC-170481.

Reply:
Changed.

Reply:
+1